core/backend/src/IModelDb.ts

```diff
@@ -2151,12 +2151,12 @@ export namespace IModelDb { // eslint-disable-line no-redeclare
 }
 
 /**
- * Argument to a function that can accept an authenticated user.
+ * Argument to a function that can accept a valid access token.
  * @public
  */
-export interface UserArg {
-  /** If present, the user's access token for the requested operation. If not present, use [[IModelHost.getAccessToken]] */
-  readonly user?: AccessToken;
+export interface TokenArg {

```

Comment:
@kabentley thoughts on the name change? With the only property in the type being an `AccessToken` it seemed better for consistency to change it and the property name to better describe it.

I went back and forth between `accessToken` and `token` for the property and `TokenArg` and `AccessTokenArg` as the interface...